src/app/FaultManager/FaultManager.h

```diff
@@ -9,31 +9,41 @@
  * Provides functionality for latching faults. Some faults are latched by default.
  */
 
+#define ERROR_DATA_LEN 7
+
+// Error type used in intermediate indexes, assessed by higher level code for fault generation.
+typedef struct
+{
+    bool active;
+    char data[ERROR_DATA_LEN];
```

Comment:
Does this array store the status of all the faults?

Reply:
nah its per-fault. Its a place to put additional data for debugging. ERROR_DATA_LEN is 7 bytes so it fits nice in the alert CAN message